pandas/tests/groupby/test_transform.py

```diff
@@ -470,7 +471,8 @@ def _check_cython_group_transform_cumulative(pd_op, np_op, dtype):
     ans = np.zeros_like(data)
 
     labels = np.array([0, 0, 0, 0], dtype=np.int64)
-    pd_op(ans, data, labels, is_datetimelike)
+    ngroups = 1
```

Comment:
Why is this hard coded to 1?

Reply:
because labels is already hard-coded to [0, 0, 0, 0], and that means there's one group